raiden/network/proxies/secret_registry.py

```diff
@@ -71,6 +75,22 @@ def __init__(self, jsonrpc_client, secret_registry_address, contract_manager: Co
     def register_secret(self, secret: Secret):
         self.register_secret_batch([secret])
 
+    def register_secret_light(self, signed_tx: SignedTransaction):
+        try:
+            log.debug("register_secret_light called")
```

Comment:
not sure if you meant to leave this `log` call here (it can be ok to do so, just checking)

Reply:
I prefer removing it. It was in the raiden's code, but I don't think it give us something useful